code/modules/virus2/effect/effect.dm

```diff
@@ -212,9 +211,79 @@
 			for(var/i = 0 to multiplier)
 				new/mob/living/simple_animal/bee(get_turf(mob))
 
+/datum/disease2/effect/hyronalinis
+	name = "Hyronalinis"
+	stage = 1
+
+/datum/disease2/effect/hyronalinis/activate(var/mob/living/carbon/mob)
+	to_chat(mob, "<span class = 'notice'>You gained radiation resistance.</span>")
+	if (prob(85))
+		mob.reagents.add_reagent(HYRONALIN, 1)
+
 
-////////////////////////STAGE 2/////////////////////////////////
+/datum/disease2/effect/soreness
+	name = "Soreness"
+	stage = 1
+
+/datum/disease2/effect/soreness/activate(var/mob/living/carbon/mob)
+	to_chat(mob, "<span class = 'notice'>You feel a little sore.</span>")
 
+/datum/disease2/effect/salutaretin
+	name = "Salutaretin Syndrome"
+	stage = 1
+
+/datum/disease2/effect/salutaretin/activate(var/mob/living/carbon/mob)
+	if(..())
+		return 1
+
+	if(mob.stat || mob.health < 90 || mob.getBrainLoss() >= 10)
```

Comment:
```suggestion
	if(mob.isUnconscious() || mob.health < 90 || mob.getBrainLoss() >= 10)
```

also checking `health` directly against a magic number is probably not what you want, especially considering there's a config option that determines health thresholds. If you just want to prevent this proc from running if the user is unconscious the following should be fine
```suggestion
	if(mob.isUnconscious() || mob.getBrainLoss() >= 10)
```